WordPress/Classes/System/ReaderWindowManager.swift

```diff
@@ -0,0 +1,37 @@
+import UIKit
+import SwiftUI
+import Combine
+import WordPressShared
+import WordPressUI
+
+class ReaderWindowManager: WindowManager {
```

Comment:
Again, probably not ideal design, but it follows `JetpackWindowManager` as an example.